docs/application/dotnet/guides/nui/quickpanelclient.md

```diff
@@ -0,0 +1,101 @@
+# Quickpanel Client
```

Comment:
We follow sentence case for titles for new documents, hence please change to:
# Quickpanel client

Reply:
Done.